generators/app/tasks/installDependencies.js

```diff
@@ -46,10 +45,10 @@ const DEV_DEPENDENCIES = [
   'prettier',
   'prettier-eslint',
   'react-hot-loader',
-  'react-app-rewire-wolox',
```

Comment:
Can u update `react-app-rewire-wolox` o create a similar lib with actual webpack config and rescripts?? 😇